LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/support/dialog-in-rtl-iframe-child-old.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html dir="rtl">
+  <head>
+    <style>
+      body {
+        /* We need a background due to https://crbug.com/1414158. */
+        background-color: white;
+        height: 200vh;
+      }
+
+      ::view-transition-old(*) {
+        animation: unset;
+        opacity: 1;
+      }
+      ::view-transition-new(*) {
+        animation-duration: 30s;
+        opacity: 0;
+      }
+
+      dialog {
+        width: 50dvw;
+        height: 50dvh;
+        box-sizing: border-box;
+        background-color: limegreen;
+        border: 1px solid black;
+      }
+
+    </style>
+    <script>
+      function show() {
+        document.getElementById('dialog').showModal();
+      }
+      function startTransition() {
+        show();
+        window.transition = document.startViewTransition(() => {
```

Comment:
```suggestion
        document.startViewTransition(() => {
```

Can probably be omitted

Reply:
The outer document uses this to decide when to take the snapshot.